releaseNotes/AzureDevOpsPipelineExtension.md

```diff
@@ -2,6 +2,10 @@
 
 The Azure App Configuration extension in Azure DevOps pipeline has been published to [Azure DevOps marketplace](https://marketplace.visualstudio.com/items?itemName=AzureAppConfiguration.azure-app-configuration-task&ssr=false#overview). The version of the Azure App Configuration task being used will print to the console when the task is executed: ![sample](pictures/AzureDevOpsExtensionVersionSample.PNG)
 
+### v2.0.10 - March, 21 2020
+* Supported to trim key prefixes before set key-values into environment variables.
```

Comment:
We could rephrase this as `Added support to trim one or more prefixes from the keys of all key-values before these are set as environment variables`

Reply:
`Added support to trim key prefixes before setting as pipeline variables.`